examples/js/atom/create-wallet.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Create a TSS atom wallet at BitGo.
+ * This makes use of the convenience function generateWallet
+ *
+ * This tool will help you see how to use the BitGo API to easily create a wallet.
+ * In this form, it creates 2 keys on the host which runs this example.
+ * It is HIGHLY RECOMMENDED that you GENERATE THE KEYS ON SEPARATE MACHINES for real money wallets!
+ *
+ * To perform more advanced features, such as encrypting keys yourself, please look at createWalletAdvanced.js
+ *
+ * Copyright 2023, BitGo, Inc.  All Rights Reserved.
+ */
+const BitGo = require('bitgo');
+const bitgo = new BitGo.BitGo({
+  env: 'custom',
+  customRootURI: '',
+});
```

Comment:
NIT: probably best use test env as this would be referred by the public and they almost never will have to use a custom URI
```suggestion
const bitgo = new BitGo.BitGo({
  env: 'test',
});
```

Reply:
sure, I will do it in next PR 
here is ticket : https://bitgoinc.atlassian.net/browse/BG-67328

Reply:
Okay